app/src/main/java/com/quran/labs/androidquran/service/AudioService.kt

```diff
@@ -1,63 +1,37 @@
-/* 
```

Comment:
this was the case a very long time ago, today, the code likely looks very different.